### ISSUE TITLE
Array with SearchObjects fails with exactly 5 SearchObjects

### ISSUE THREAD

**Issue by alex-lechner** [andris9/imapflow#50]
**Describe the bug**
I fetch emails based on 5 SearchObjects but the query fails. Less than 5 and more than 5 are working though.

The error: `1E BAD Error in IMAP command UID SEARCH: Missing argument (0.001 + 0.000 secs).`
The query from ImapFlow: `1E UID SEARCH OR OR OR TO mail1@domain.com TO mail2@domain.com OR TO mail3@domain.com TO mail4@domain.com OR TO mail5@domain.com`

**To Reproduce**
```js
const client = new ImapFlow({ /* config here */ });
const SERACH_OBJS: Object[]  = [
  { to: 'mail1@domain.com' },
  { to: 'mail2@domain.com' },
  { to: 'mail3@domain.com' },
  { to: 'mail4@domain.com' },
  { to: 'mail5@domain.com' },
]

async function fetchMails() {
  await client.connect();
  let lock = await client.getMailboxLock("INBOX");
  try {
    for await (let message of client.fetch(
        { or: SERACH_OBJS }, /* pass SEARCH_OBS here */
        { envelope: true, uid: true, emailId: true }, /* Another bug: emailID is not fetched! */
        { uid: true }
    )) {
      /* DO STUFF HERE */
    }
  } finally {
    lock.release();
  }
 await client.logout();
}
```

**Expected behavior**
Return FetchMessageObject

**Comment by andris9** [comment]
Can confirm, it's a bug, there's an extra `OR` before `mail5@domain.com`. Will look into it

**Comment by andris9** [comment]
Fixed in v1.0.59
